documentation/query-language.html

```diff
@@ -329,6 +329,58 @@ <h3 id="near-and-ordered-near">Near and Ordered Near</h3>
 </p>
 
 
+<h3 id="sameElement">Search within same struct element</h3>
+<p>
+<em>sameElement()</em> is an operator that requires the terms to match within the same struct element in an array or a map field. Example:
+</p>
+<pre>
+...
+struct person {
+  field first_name type string {}
+  field last_name type string {}
+  field year_of_birth type int {}
+}
+
+field persons type array&lt;person&gt; {
+  indexing: summary
+  struct-field first_name { indexing: attribute }
+  struct-field last_name { indexing: attribute }
+  struct-field year_of_birth { indexing: attribute }
+}
+field identities type map&lt;string, person&gt; {
+  indexing: summary
+  struct-field key { indexing: attribute }
+  struct-field value.first_name { indexing: attribute }
+  struct-field value.last_name { indexing: attribute }
+  struct-field value.year_of_birth { indexing: attribute }
+}
+...
+</pre>
+<p>
+With normal <em>AND</em> the query <pre>persons.first_name AND persons.last_name</pre> will normally not give you what you want.
```

Comment:
Is this expected to be inline and not end up as its own paragraph? If yes, it should be `<code>persons.first_name AND persons.last_name</code>` as that is displayed inline while `<pre>` is a block (like a `<div>` or a `<p>`). Alternatively, if it's meant to be a block, close the open paragraph element before it and open a new one after it.